GOVERNANCE.md

```diff
@@ -129,7 +129,16 @@ When an agenda item has appeared to reach a consensus the moderator
 will ask "Does anyone object?" as a final call for dissent from the
 consensus.
 
+Agenda items should be tagged at least 24 hours in advance. If an item
+is added less than 24 hours before the meeting it cannot be brought to
+a vote.
+
 If an agenda item cannot reach a consensus a TC member can call for
 either a closing vote or a vote to table the issue to the next
 meeting. The call for a vote must be seconded by a majority of the TC
-or else the discussion will continue. Simple majority wins.
+or else the discussion will continue. Simple majority wins, those not
+in attendance abstain.
```

Comment:
That's quite reasonable so long as there is an expected time frame within
which such emails can be expected.
On Mar 20, 2015 3:52 PM, "Trevor Norris" <notifications@github.com> wrote:

> In GOVERNANCE.md
> <https://github.com/iojs/io.js/pull/1222#discussion_r26882797>:
>
> >  If an agenda item cannot reach a consensus a TC member can call for
> >  either a closing vote or a vote to table the issue to the next
> >  meeting. The call for a vote must be seconded by a majority of the TC
> > -or else the discussion will continue. Simple majority wins.
> > +or else the discussion will continue. Simple majority wins, those not
> > +in attendance abstain.
>
> Disagree. We should be able to email in our vote if circumstances don't
> allow us to attend.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/iojs/io.js/pull/1222/files#r26882797>.
>